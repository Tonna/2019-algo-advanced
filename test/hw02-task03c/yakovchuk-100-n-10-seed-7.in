10
9 4 2 5 8 1 10 7 3 6
