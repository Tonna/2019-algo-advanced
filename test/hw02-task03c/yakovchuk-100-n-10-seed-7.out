9 8 5 7 9 8 9 10 9 10
